scale/ingest/triggers/ingest_recipe_handler.py

```diff
@@ -81,6 +85,11 @@ def process_ingested_source_file(self, ingest_id, source, source_file, when):
         if recipe_revision:
             recipe_type = RecipeTypeRevision.objects.get_revision(recipe_name, recipe_revision).recipe_type
             
+        if len(recipe_type.get_definition().get_input_keys()) == 0:
+            logger.info('No inputs defined for recipe %s. Recipe will not be ran.' % recipe_name)
```

Comment:
"Recipe will not be run." I know, I'm pedantic.

Reply:
"Run, recipe shall not" got it.